aem/aem_skyline_install.md

```diff
@@ -0,0 +1,98 @@
+
+## Integrate with AEM as a Cloud Service
+
+This documentation is specific to `AEM as a Cloud Service` set up, to integrate with `AEM on premise` 
+please refer to the other associated [documentation](aem_on_premise_install.md).
```

Comment:
Shall we break this statement into two separate statements?
```suggestion
This documentation is specific to `AEM as a Cloud Service` set up.
To integrate with `AEM on premise`, please refer to the other associated [documentation](aem_on_premise_install.md).
```

Reply:
done